### PR TITLE
fix(combobox): long text truncates on single and single-persist modes

### PR DESCRIPTION
**Pull request by Elijbet**
**Related Issue:** #7020

## Summary

The `close` button no longer overlaps a populated combobox in `single` and `single-persist` modes.

### CODE REVIEW

File: packages/calcite-components/src/components/combobox/combobox.stories.ts [hunk 1]
```diff
@@ -333,21 +333,64 @@ export const nestedItems = (): string => html`
...
+    calcite-combobox {
+      width: 260px;
+    }
+    calcite-combobox-item {
+      width: 260px;
+    }
+  </style>
+`;
+
+export const longItemsSingle = (): string => html`
```
Comment (jcfranco) [inline packages/calcite-components/src/components/combobox/combobox.stories.ts:347]: Can these be consolidated into a `longItemsAllSelectionModes` (or similar) story?

**Review by jcfranco — APPROVED** [review]
@eriklharper, @ashetland could help review this one?

Code changes and screenshot changes LGTM! This should be good to merge once additional reviews are ✅.

**Comment by SkyeSeitz** [comment]
Changes lg2m! Thanks, Eliza

File: packages/calcite-components/src/components/combobox/combobox.stories.ts [hunk 2]
```diff
@@ -333,21 +333,56 @@ export const nestedItems = (): string => html`
...
+    }
+    calcite-combobox-item {
+      width: 260px;
+    }
+  </style>
+`;
+
+export const longItemsAllSelectionModes = (): string => html`
+  ${style}
+  <div style="display: flex; flex-direction: column;">
```
Comment (macandcheese) [inline packages/calcite-components/src/components/combobox/combobox.stories.ts:349]: These render on top of one another in the Chromatic screenshots, can we separate the with css or into different stories?
Screenshot 2024-02-18 at 2 16 57 PM

**Comment by macandcheese** [comment]
Changes look good, I think I agree this is low-risk @jcfranco

**Comment by eriklharper** [comment]
LGTM!  Sorry for the late review, but just checked it out on `main` and it's SMOKIN! 

image

**Pull request merged**